OpenDreamRuntime/Objects/MetaObjects/DreamMetaObjectWorld.cs

```diff
@@ -38,7 +47,7 @@ sealed class DreamMetaObjectWorld : IDreamMetaObject {
             }
 
             DreamValue view = dreamObject.ObjectDefinition.Variables["view"];
-            if (view.TryGetValueAsString(out string viewString)) {
+            if (view.TryGetValueAsString(out var viewString) && viewString != null) {
```

Comment:
`TryGetValueAsString()` should never give you null if it returns true.
```suggestion
            if (view.TryGetValueAsString(out var viewString)) {
```